portal-wire-protocol.md

```diff
@@ -350,6 +350,7 @@ The process works as follows:
 
 - A DHT node is offered and receives a piece of content that it is interested in.
 - This DHT node checks their routing table for `k` nearby DHT nodes that should also be interested in the content.
+- Those `k` nodes should not include the node that originally provided aformentioned piece of content.
```

Comment:
I think we should (or rather MUST :-)) always use `SHOULD` in caps as per https://datatracker.ietf.org/doc/html/rfc2119

I also think this note can go with the previous point, as it is functionally part of that step.